dwitter/templates/base.html

```diff
@@ -29,6 +29,8 @@
     <link rel=stylesheet href='{% static "main.css" %}' >
     {% endcompress %}
 
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
```

Comment:
Would it be better to host this locally? I prefer to avoid external resources, especially nice for offline work on airplanes (which I seem to do surprisingly frequently)

Reply:
Probably a good idea